catalog/pages/inputs/HybridSelectExample.js

```diff
@@ -1,121 +1,61 @@
-// this components was created for hybrid dropdown demonstration purpose
-import React, { Component } from "react";
-import styled from "styled-components";
+import React, { Component, Fragment } from "react";
 import {
-  Select,
-  DropDownGroup,
-  DropDownOption,
-  DropDownLabel
+  DropDownLabel,
+  HybridSelect,
+  HybridOption
 } from "../../../src/components/Input";
 
-const HybridSelectWrapper = styled.div`
-  position: relative;
-`;
-
 export default class HybridSelectExample extends Component {
   state = {
-    showNativeSelect: true,
-    selectedValue: [],
-    isOpen: false,
-    mouseOnDropdown: false
+    value: []
   };
 
-  // onChange function called when value of dropdown is changed
   onChange = value => {
-    this.setState({ selectedValue: value });
-  };
-
-  // update function called when value of native select is changed
-  updateValue = e => {
-    this.setState({ selectedValue: [e.target.value] }); // saving value in array to match dropdown's format
-  };
-
-  // To show native select
-  showNativeSelect = () => {
-    // Updating value when we know mouse left dropdown
-    this.setState({ mouseOnDropdown: false });
-    if (!this.state.isOpen) {
-      this.setState({ showNativeSelect: true });
-    }
-  };
-
-  // To hide native select - called when mouse enters native select
-  hideNativeSelect = () => {
-    this.setState({ showNativeSelect: false, mouseOnDropdown: true });
-  };
-
-  // Set isOpen state to true when dropdown menu opens
-  dropdownMenuOpen = () => {
-    this.setState({ isOpen: true });
-  };
-
-  // Set isOpen state to false when dropdown menu closes
-  dropdownMenuClose = () => {
-    this.setState({ isOpen: false }, () => {
-      if (!this.state.mouseOnDropdown) {
-        this.showNativeSelect();
-      }
-    });
+    this.setState({ value });
   };
 
   render() {
-    const { showNativeSelect, selectedValue } = this.state;
+    const { value } = this.state;
     return (
-      <HybridSelectWrapper>
-        <DropDownLabel id="selectLabel" htmlFor="selectElement" size="large">
+      <Fragment>
```

Comment:
There is a new, shorter syntax you can use for declaring fragments. It looks like empty tags:

```
      <>
        <td>Hello</td>
      </>
```

Reply:
Thanks. Updated.